subseas_flex_fcst/layout.py

```diff
@@ -228,7 +231,80 @@ def navbar_layout(phys_units):
                 ],
                 id="threshold_style"
             ),
-            dbc.Alert(
+            html.Div(
+                [
+                    help_layout(
+                        "Start Date",
+                        "start_date",
+                        "Start date choices",
+                    ),
+                ],
+                style={
+                    "color": "white",
+                    "position": "relative",
+                    "width": "105px",
+                    "display": "inline-block",
+                    "padding": "10px",
+                    "vertical-align": "top",
+                }
+            ),            
+            html.Div(
+                [
+                    dcc.Dropdown(
+                        id="startDate",
+                        clearable=False,
+                        options=[
+                            dict(label="Apr 1", value="Apr-1-2022"),
+                            dict(label="Apr 4", value="Apr-4-2022"),
+                            dict(label="Apr 6", value="Apr-6-2022"),
+                            dict(label="Apr 8", value="Apr-8-2022"),
+                            dict(label="Apr 11", value="Apr-11-2022"),
+                            dict(label="Apr 13", value="Apr-13-2022"),
+                            dict(label="Apr 15", value="Apr-15-2022"),
+                            dict(label="Apr 18", value="Apr-18-2022"),
+                            dict(label="Apr 20", value="Apr-20-2022"),
+                            dict(label="Apr 22", value="Apr-22-2022"),
+                            dict(label="Apr 25", value="Apr-25-2022"),
+                            dict(label="Apr 27", value="Apr-27-2022"),
+                            dict(label="Apr 29", value="Apr-29-2022"),
+                        ],
+                        value="Apr-1-2022", 
+                    ),
+                ],style={"width":"6%"},
+            ),
+            html.Div(
+                [
+                    help_layout(
+                        "Lead time",
+                        "lead_time",
+                        "Lead time range choices, in weeks from the start date",
```

Comment:
And would the target date be the target_start value?

Reply:
It would ideally be target_start - target_end appended, e.g. 2 Apr - 8 Apr 2022. This means this control needs to be a callback dependent on the choice of S, at least for its labels, as its values can remain 1, 2, 3, 4.

Reply:
will be addressed in separate issue.